ferrocene/doc/user-manual/src/rustfmt/install.rst

```diff
@@ -0,0 +1,37 @@
+.. SPDX-License-Identifier: MIT OR Apache-2.0
+   SPDX-FileCopyrightText: The Ferrocene Developers
+
+Installing ``rustfmt``
+======================
+
+1. Download the ``rustfmt`` archive for your host platform
+
+   The pattern of the name is:
+
+   .. code-block::
+
+      rustfmt-<target>-<hash>.tar.xz
+
+   For example for the ``nightly-2024-04-30`` release of the ``x86_64-unknown-linux-gnu`` target it is:
+
+   .. code-block::
+
+      rustfmt-x86_64-unknown-linux-gnu-af9a431b1.tar.xz
+
+2. Unpack the archive
+
+   .. note::
+
+      It is recommended to unpack it to the same directory as the rest of the Ferrocene toolchain, but it is not strictly neccessary.
```

Comment:
This is wrong, `rustfmt` requires rustc to be installed in the same directory, as it dynamically links to it.

Reply:
Oh really? I did not know.

Is this Ferrocene specific or does rustfmt always require to be placed in the same dir as rustc?

Reply:
You always need to install rustfmt in the same directory as rustc. If you run `ldd` on a rustfmt binary you see it dynamically links to `librustc_driver.so`, which is provided by rustc.